similarity/.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: false
```

Comment:
Nit: this was one of the files I was saying probably doesn't make sense to include. It's specific to Travis CI, which you likely won't be using for the project.

The requirements.txt should be sufficient in sharing your dependencies with the rest of the pod.